package.json

```diff
@@ -47,13 +47,14 @@
     "grunt-contrib-connect": "^0.9.0",
     "grunt-contrib-copy": "^0.7.0",
     "grunt-contrib-jshint": "^0.10.0",
-    "grunt-contrib-uglify": "^0.6.0",
+    "grunt-contrib-uglify": "^0.7.0",
     "grunt-extract-cldr-data": "^1.1.0",
-    "grunt-saucelabs": "~8.3.2",
+    "grunt-json-remove-fields": "git://github.com/ericf/grunt-json-remove-fields#improve-files-and-output",
```

Comment:
Waiting on https://github.com/diepet/grunt-json-remove-fields/pull/1 to make this a proper npm registry dep.